kurtosis-devnet/tests/interop/boilerplate_test.go

```diff
@@ -1,10 +1,12 @@
 package interop
 
 import (
-	"log/slog"
 	"os"
+
+	oplog "github.com/ethereum-optimism/optimism/op-service/log"
+	"github.com/ethereum/go-ethereum/log"
 )
 
 func init() {
-	slog.SetDefault(slog.New(slog.NewTextHandler(os.Stderr, &slog.HandlerOptions{Level: slog.LevelDebug})))
+	oplog.SetGlobalLogHandler(log.NewTerminalHandlerWithLevel(os.Stderr, log.LevelDebug, true))
```

Comment:
This is fine for now, but it's best if the existing devnet-SDK utils start to use logger instances instead of the global logger. The global logger doesn't get filtered to specific tests, and misses the useful filename prefix etc. cc @sigma